styles/typography.ts

```diff
@@ -67,6 +69,7 @@ export const SubTitle = styled.h3<TextType>`
   color: ${(props: TextType) => props.color ? props.color : "black"};
   font-size: ${(props: TextType) =>
       props.size ? props.size : "2rem"};
+  font-weight: 1000;
```

Comment:
You added a weight prop in the interface but then only used it in body, add it here too so that it's set up to be modular in the future and have the weight you hard coded be the default if there's no prop passed

Reply:
This is the comment that made the review amazing

Reply:
Glad to hear it but the comment wasn't implemented lol